auto-configurations/mcp/spring-ai-autoconfigure-mcp-client-common/src/main/java/org/springframework/ai/mcp/client/common/autoconfigure/properties/McpStreamableHttpClientProperties.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2025-2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.ai.mcp.client.common.autoconfigure.properties;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties for Streamable Http client connections.
+ *
+ * <p>
+ * These properties allow configuration of multiple named Streamable Http connections to
+ * MCP servers. Each connection is configured with a URL endpoint for communication.
+ *
+ * <p>
+ * Example configuration: <pre>
+ * spring.ai.mcp.client.streamable:
+ *   connections:
+ *     server1:
+ *       url: http://localhost:8080/events
+ *     server2:
+ *       url: http://otherserver:8081/events
+ * </pre>
+ *
+ * @author Christian Tzolov
+ * @see ConnectionParameters
+ */
+@ConfigurationProperties(McpStreamableHttpClientProperties.CONFIG_PREFIX)
+public class McpStreamableHttpClientProperties {
+
+	public static final String CONFIG_PREFIX = "spring.ai.mcp.client.streamable";
```

Comment:
Perhaps the proper prefix should be `spring.ai.mcp.client.streamable-http`?

Reply:
Then the class name should be `McpStreamableHttpHttpClientProperties`

Reply:
yeah, this is better!